src/karen.lisp

```diff
@@ -4,3 +4,7 @@
 (in-package :karen)
 
 ;; blah blah blah.
+
+;; expect some linter messages...
+(let ((foo "unused"))
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
style-warning: The variable FOO is defined but never used.